Application/Platform/Gatekeeper/Authentication/Frontend.php

```diff
@@ -322,20 +326,116 @@ public function frontendIdentificationCredential($CredentialName = null, $Creden
                     ),
                     new FormRow(
                         new FormColumn(array(
-                            (new Primary('Anmelden')),
+                            (new Primary('Login')),
                         ))
                     )
                 )
             )
         );
 
+        // set depending information
+        if(strtolower($this->getRequest()->getHost()) == 'www.schulsoftware.schule'
+//            || $this->getRequest()->getHost() == '192.168.75.128' // local test
+        ){
+            $Form.= new Layout(new LayoutGroup(new LayoutRow(
+                new LayoutColumn(array(
+                    '<br/><br/><br/><br/>',
+                    new Title('Anmeldung CONNEXION (Pilot)'),
+                    new PrimaryLink('Login', 'SPHERE\Application\Platform\Gatekeeper\Saml\Login\EVSSN')
+                    //. new Link('.', 'SPHERE\Application\Platform\Gatekeeper\Saml\Login\EKM') // EKM -> Beispiel kann für zukünftige IDP's verwendet werden
+                ))
+            )));
+        }
+
         setcookie('cookies_available', 'enabled', time() + (86400 * 365), '/');
 
         $View->setContent($this->getIdentificationLayout($Form));
 
         return $View;
     }
 
+    /**
+     * @return Stage
+     */
+    public function frontendIdentificationSamlEVSSN()
+    {
+
+        return $this->LoginSecondPageLogic(SamlEVSSN::getSAML());
+    }
+
+//    /**
+//     * // EKM -> Beispiel kann für zukünftige IDP's verwendet werden
+//     * @return Stage
+//     */
+//    public function frontendIdentificationSamlEKM()
+//    {
+//
+//        return $this->LoginSecondPageLogic(SamlEKM::getSAML());
+//    }
+
+    private function LoginSecondPageLogic($Config = array())
```

Comment:
Codacy found an issue: [The method LoginSecondPageLogic() has an NPath complexity of 1728. The configured NPath complexity threshold is 200.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)